data/lanceme-up.json

```diff
@@ -1,8 +1,10 @@
 {
   "name": "Lanceme Up",
   "establishedAt": 1599978360,
-  "location": "महापाल रोड, Lalitpur",
+  "location": "Mahapal Road, Lalitpur",
```

Comment:
NOTE: Changed to english to be consistent

Reply:
GG